aws_cloudtrail_rules/aws_root_failed_console_login.yml

```diff
@@ -1,8 +1,8 @@
 AnalysisType: rule
 Filename: aws_root_failed_console_login.py
 RuleID: AWS.CloudTrail.RootFailedConsoleLogin
-DisplayName: Root Account Failed Console Login
-Enabled: true
+DisplayName: [DEPRECATED] Root Account Failed Console Login
```

Comment:
What should our pattern here be? We want to delete them eventually

Reply:
Sadly today we don't have a way to push deletes up, so if we don't do this the rule will stay present & enabled forever. We've discussed adding functionality so that the panther_analysis_tool or bulk upload button has a "sync" setting which deletes anything that wasn't included in the bulk upload, but we kind of backburnered it since we had planned to do a more holistic re-work with managed packs.

For today I say we keep it like this, then re-evaluate once we have packs implemented.